modules/cpmso-yaml-provider-spec-gcp.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-gcp_{context}"]
+= Sample GCP provider specification
+
+When you create a control plane machine set for an existing cluster, the provider specification must match the `providerSpec` configuration in the control plane machine custom resource (CR) that is created by the installation program. You can omit any field that is set in the failure domain section of the CR.
+
+In the following example, `<cluster_id>` is the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI installed, you can obtain the infrastructure ID by running the following command:
+
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+.Sample GCP `providerSpec` values
+[source,yaml]
+----
+providerSpec:
+  value:
+    apiVersion: machine.openshift.io/v1beta1
+    canIPForward: false
+    credentialsSecret:
+      name: gcp-cloud-credentials <1>
+    deletionProtection: false
+    disks:
+    - autoDelete: true
+      boot: true
+      image: <path_to_image> <2>
+      labels: null
+      sizeGb: 200
+      type: pd-ssd
+    kind: GCPMachineProviderSpec <3>
+    machineType: e2-standard-4
+    metadata:
+      creationTimestamp: null
+    metadataServiceOptions: {}
+    networkInterfaces:
+    - network: <cluster_id>-network
+      subnetwork: <cluster_id>-master-subnet
+    projectID: <project_name> <4>
+    region: <region> <5>
+    serviceAccounts:
+    - email: <cluster_id>-m@<project_name>.iam.gserviceaccount.com
+      scopes:
+      - https://www.googleapis.com/auth/cloud-platform
+    shieldedInstanceConfig: {}
+    tags:
+    - <cluster_id>-master
+    targetPools:
+    - <cluster_id>-api
+    userDataSecret:
+      name: master-user-data <6>
+    zone: "" <7>
+----
+<1> Specifies the secret name for the cluster. Do not change this value.
+<2> Specifies the path to the image that was used to create the disk. If you have the OpenShift CLI installed, you can obtain the path to the image by running the following command:
```

Comment:
in the preview it looks like text is overlapping , see if this can be fixed , i am not sure if this is just on preview ..

![image](https://user-images.githubusercontent.com/59862091/221860859-266a3522-f0ae-46cd-bff2-646460889ed7.png)

Reply:
hm, yes looks like the command box overrides the margin instead of scrolling here (usually long commands create scrollbars). I will see if there is anything to be done about it - it might be necessary to break the second line to resolve this.